ConsoleCalculator.Tests/CalculatorFixture.cs

```diff
@@ -5,10 +5,79 @@ namespace ConsoleCalculator.Tests
 {
     public class CalculatorFixture
     {
+        Calculator calc = new Calculator();
+
+         internal string ManykeysPressed(char[] key)
+        {
+            int i;
+            for (i = 0; i < key.Length - 1; i++)
+            {
+                calc.SendKeyPress(key[i]);
+            }
+                return calc.SendKeyPress(key[i]);
+                 
+        }
+        
+        [Fact]
+        public void Add()
+        {
+            
+            char[] key = { '1', '3', '+', '8','='};
+            var ans = ManykeysPressed(key);
+            Assert.Equal("21", ans);
+        }
+
+        
+        [Fact]
+        public void Divideby0()
+        {
+            
+            char[] key = { '1', '3', '/', '0','=' };
+            var ans = ManykeysPressed(key);
+            Assert.Equal("E", ans);
+        }
+
+        
+        [Fact]
+        public void DotOperator()

```

Comment:
Test DotOperator does not signify what it is trying to verify, better name would be TestIgnoreMultipleDecimalPoints()
Add meaningful names to all other tests